packages/authentication/README.md

```diff
@@ -20,18 +20,27 @@ npm install --save @loopback/authentication
 Start by decorating your controller methods with `@authenticate` to require
 the request to be authenticated.
 
+In this example, we make the user profile available via dependency injection
+using a key available from `@loopback/authentication` package.
+
 ```ts
-// controllers/my-controller.ts
-import {UserProfile, authenticate, AuthenticationBindings} from '@loopback/authentication';
-import {inject} from '@loopback/core';
-import {get} from '@loopback/rest';
+// src/controllers/who-am-i.controller.ts
+import {inject} from '@loopback/context';
```

Comment:
I think importing `inject` from `@loopback/core` is fine as that abstracts `@loopback/context`. But this is fine too. I think we should decide what style we want to advocate. (Don't change this until there's some more feedback from the team).

I personally think it's better to abstract `@loopback/context` from new to intermediate app developers. Only Advanced users + extension developers would care enough to dig into the context module imo and exposing `context` in such README's / examples might lead them to dig futher.

Reply:
+1 for importing from `@loopback/core`.